automation_tools/__init__.py

```diff
@@ -1893,23 +2048,87 @@ def satellite6_upgrade(admin_password=None):
     run('katello-service status')
 
 
-def product_upgrade(product, instance_name, image_name, flavor_name, ssh_key):
+def capsule6_upgrade(admin_password=None):
+    """Upgrades capsule from already created Openstack image
+    of old capsule version to latest capsule CDN/compose version.
+
+    :param admin_password: A string. Defaults to 'changeme'.
+        Foreman admin password for hammer commands.
+
+    The following environment variables affect this command:
+
+    ADMIN_PASSWORD
+        Optional, defaults to 'changeme'. Foreman admin password.
+    CAPSULE_URL
+        Optional, defaults to available capsule version in CDN.
+        URL for capsule of latest compose to upgrade.
+
+    """
+    if admin_password is None:
+        admin_password = os.environ.get('ADMIN_PASSWORD', 'changeme')
+    # Removing rhel-released and rhel-optional repo
+    # run('rm -rf /etc/yum.repos.d/rhel-{optional,released}.repo')
+    # Update the packages
+    print('Wait till Packages update ... ')
+    update_packages(quiet=True)
+    # Setting Capsule61 Repos
+    major_ver = distro_info()[1]
+    if os.environ.get('CAPSULE_URL') is None:
+        enable_repos('rhel-{0}-server-satellite-capsule-6.1-rpms'.format(
+            major_ver))
+    disable_repos('rhel-{0}-server-satellite-capsule-6.0-rpms'.format(
+        major_ver))
+    # Stop katello services, except mongod
+    run('for i in qpidd pulp_workers pulp_celerybeat '
+        'pulp_resource_manager httpd; do service $i stop; done')
+    # yum cleaning all
```

Comment:
I have added those comments just to separate the each step in code. Still I can remove few comments as those are not necessary to be there.